.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 .vscode/
+Cargo.lock
```

Comment:
`Cargo.lock` needs to be included in the git repository for the C API functionality with meson.